src/features/pastelDB/pastelDBLib.ts

```diff
@@ -154,78 +164,89 @@ export const insertStatisticDataToDB = (
   hashrate: number,
   difficulty: number,
 ): void => {
-  const create_timestamp: number = +new Date()
+  const createTimestamp: number = +new Date()
   const newId = getLastIdFromDB(pastelDB, 'statisticinfo')
-  const sqlText = `INSERT INTO statisticinfo
-      (id, hashrate, miner_distribution, difficulty, create_timestamp) VALUES
-      (${newId}, '${hashrate}', '', '${difficulty}', ${create_timestamp})`
-  pastelDB.exec(sqlText)
+  const values = {
+    $newId: newId,
+    $hashrate: hashrate,
+    $difficulty: difficulty,
+    $createTimestamp: createTimestamp,
+  }
+
+  pastelDB.exec(insertStatisticinfoQuery, values)
 }
 
 export const getDatasFromDB = (
   pastelDB: Database,
   tableName: string,
 ): QueryExecResult[] => {
-  const sqlText = `SELECT * FROM ${tableName}`
+  if (tableNames[tableName] !== true) {
+    throw new Error('pastelDB getDatasFromDB error: table name is invalid')
+  }
+
+  const sqlText = selectAllQuery + tableName
   const sqlResult = pastelDB.exec(sqlText)
-  console.log(`${tableName} table data: `, sqlResult)
   return sqlResult
 }
 
 export const insertNetworkInfotoDB = (
   pastelDB: Database,
   networkinfo: TNetworkInfo,
 ): void => {
-  const create_timestamp: string = new Date().toLocaleTimeString()
+  const createTimestamp: string = new Date().toLocaleTimeString()
   const newId = getLastIdFromDB(pastelDB, 'networkinfo')
   const networks = JSON.stringify(networkinfo.networks)
-  const sqlText = `INSERT INTO networkinfo VALUES (
-    ${newId},
-    '${networkinfo.version}',
-    '${networkinfo.subversion}',
-    '${networkinfo.protocolversion}',
-    '${networkinfo.localservices}',
-    '${networkinfo.timeoffset}',
-    '${networkinfo.connections}',
-    '${networks}',
-    '${networkinfo.relayfee}',
-    '${networkinfo.localaddresses}',
-    '${networkinfo.warnings}',
-    '${create_timestamp}'
-  )`
-  pastelDB.exec(sqlText)
+  const localaddresses = JSON.stringify(networkinfo.localaddresses)
+  const values = {
+    $newId: newId,
+    $version: networkinfo.version,
+    $subversion: networkinfo.subversion,
+    $protocolversion: networkinfo.protocolversion,
+    $localservices: networkinfo.localservices,
+    $timeoffset: networkinfo.timeoffset,
+    $connections: networkinfo.connections,
+    $networks: networks,
+    $relayfee: networkinfo.relayfee,
+    $localaddresses: localaddresses,
+    $warnings: networkinfo.warnings,
+    $createTimestamp: createTimestamp,
+  }
+
+  pastelDB.exec(insertNetworkinfoQuery, values)
 }
 
 export const insertNetTotalsToDB = (
   pastelDB: Database,
   nettotals: TNetTotals,
 ): void => {
-  const create_timestamp: string = new Date().toLocaleTimeString()
+  const createTimestamp: string = new Date().toLocaleTimeString()
```

Comment:
Why is this createTimestamp const using `toLocaleTimeString()` and others are not?